src/osd/OSDMap.cc

```diff
@@ -3112,7 +3113,8 @@ class OSDTreePlainDumper : public CrushTreeDumper::Dumper<TextTable> {
       } else {
 	*tbl << (osdmap->is_up(qi.id) ? "up" : "down")
```

Comment:
destroyed shoudl imply down. (maybe reverify that the destroy and purge commands fail is the osd i sup, and that prepare_boot also fails if the osd is marked destroyed?)